lib/music/MusicManager.js

```diff
@@ -1,18 +1,36 @@
-const { Collection, Client } = require("discord.js"); // eslint-disable-line no-unused-vars
-const LavalinkClient = require("./LavalinkClient.js");
+const Discord = require("discord.js");
```

Comment:
yet, this should be removed and music interface should just be used

Reply:
Lavalink client it attached to client as well as this

Reply:
okay? that isnt what im saying